tests/test_particleContainer.py

```diff
@@ -40,6 +40,25 @@ def particle_container(Npart, std_geometry, distmap, boxarr, std_real_box):
 
     iseed = 1
     pc.InitRandom(Npart, iseed, myt, False, std_real_box)
+
+    # add runtime components: 1 real 2 int
+    pc.add_real_comp(True)
+    pc.add_int_comp(True)
+    pc.add_int_comp(True)
+
+    #   can be removed after
+    #     https://github.com/AMReX-Codes/amrex/pull/3615
+    pc.resize_runtime_real_comp(1, True)
+    pc.resize_runtime_int_comp(2, True)
+
+    # assign some values to runtime components
+    for lvl in range(pc.finest_level + 1):
+        for pti in pc.iterator(pc, level=lvl):
+            soa = pti.soa()
+            soa.GetRealData(2).assign(1.2345)
```

Comment:
When we call assign here, are they already properly resized (from `resize_runtime_real_comp`)?

Reply:
They were not: https://github.com/AMReX-Codes/amrex/pull/3615